cpp/src/request_tag.cpp

```diff
@@ -53,12 +62,13 @@ std::shared_ptr<RequestTag> createRequestTag(
   return req;
 }
 
-RequestTag::RequestTag(std::shared_ptr<Component> endpointOrWorker,
-                       const std::variant<data::TagSend, data::TagReceive> requestData,
-                       const std::string operationName,
-                       const bool enablePythonFuture,
-                       RequestCallbackUserFunction callbackFunction,
-                       RequestCallbackUserData callbackData)
+RequestTag::RequestTag(
+  std::shared_ptr<Component> endpointOrWorker,
+  const std::variant<data::TagSend, data::TagReceive, data::TagReceiveWithHandle> requestData,
+  const std::string operationName,
```

Comment:
Nit. `const string` would cause an unnecessary copy IMP. How about pass-by-value and move?